llvm/test/CodeGen/RISCV/fold-addi-loadstore.ll

```diff
@@ -733,3 +733,270 @@ entry:
   %0 = load i32, ptr inttoptr (i64 2147481600 to ptr)
   ret i32 %0
 }
+
+%struct.S = type { i64, i64 }
+
+define dso_local i64 @fold_addi_from_different_bb(i64 %k, i64 %n, ptr %a) {
```

Comment:
Drop `dso_local`. Add `nowunwind` so the cfi directives will go away.

Reply:
Done.